src/ServicePulse.Host/vue/src/components/monitoring/LargeGraph.vue

```diff
@@ -0,0 +1,176 @@
+const series1 = useGraph(
+  () => props.firstdataseries,
+  () => props.minimumyaxis,
+  () => minPoints.value
+);
+const series2 = useGraph(
+  () => props.seconddataseries,
+  () => props.minimumyaxis,
+  () => minPoints.value
+);
+const maxYaxis = computed(() => padToWholeValue(Math.max(series1.maxYaxis.value, series2?.maxYaxis.value ?? 0)));
+const tickValues = computed(() => {
+  const ticks = [0, (maxYaxis.value * 1) / 4, (maxYaxis.value * 1) / 2, (maxYaxis.value * 3) / 4, maxYaxis.value];
+  const durationTick = (tick: number) => {
+    const formattedTime = useFormatTime(tick.toString());
+    return `${formattedTime.value} ${formattedTime.unit}`;
+  };
+  return props.isdurationgraph ? ticks.map((tick) => durationTick(tick)) : ticks;
+});
+
+function padToWholeValue(value: number) {
+  const emptyDataSetyAxisMax = 10;
+
+  if (!value) {
+    return emptyDataSetyAxisMax;
+  }
+
+  let upperBound = 10;
+
+  while (value > upperBound) {
+    upperBound *= 10;
+  }
+
+  upperBound /= 10;
+
+  return Math.floor(value / upperBound) * upperBound + upperBound;
+}
+
+const series1AverageLabelValue = computed(() => {
+  return props.isdurationgraph ? useFormatTime(series1.average.value.toString()).value : useFormatLargeNumber(series1.average.value.toString(), 2);
+});
+const series2AverageLabelValue = computed(() => {
+  return props.isdurationgraph ? useFormatTime(series2.average.value.toString()).value : useFormatLargeNumber(series2.average.value.toString(), 2);
+});
+const series1AverageLabelSuffix = computed(() => (props.isdurationgraph ? useFormatTime(series1.average.value.toString()).unit.toUpperCase() : props.metricsuffix ?? ""));
+const series2AverageLabelSuffix = computed(() => (props.isdurationgraph ? useFormatTime(series2.average.value.toString()).unit.toUpperCase() : props.metricsuffix ?? ""));
+//NOTE: using hard coded height of graph (200px - 10 for padding). To get it exact without hard coding a height value, we would need to perform measurement on the rendered SVG element, which we want to avoid
+const series1AverageLabelPosition = computed(() => `calc(${(series1.average.value / maxYaxis.value) * 190}px - 1em)`);
+const series2AverageLabelPosition = computed(() => `calc(${(series2.average.value / maxYaxis.value) * 190}px - 1em)`);
+</script>
+
+<template>
+  <div class="graph large-graph pull-left" :class="[hover ? 'hover' : '']" @mouseover="hover = true" @mouseout="hover = false">
+    <div class="padding">
+      <svg class="y-axis">
+        <!-- 
+          Offsets explained:
+            - 60px inset from the left for the y-axis values to show
+            - 5px padding above and below to prevent the bottom and top y-axis value from being cut off by the edge of the SVG 
+              (font size 10px for class 'tick', offset half a character = 5px)
+            - transform 60px from the left and 5px down for the same reasons as above
+        -->
```

Comment:
I was worried about these getting rendered in the final output, but they aren't :) 
[Vue strips them out of the production build](https://vuejs.org/api/application.html#app-config-compileroptions-comments).